apps/clock/style/alarm.css

```diff
@@ -75,6 +75,10 @@ ul.compact li label:active {
   display: none;
 }
 
+#time-select {
+  text-indent: 110%;
```

Comment:
would you mind adding a comment explaining why the `text-indent` is needed and also fixing the calendar app? thanks!